package.json

```diff
@@ -2,13 +2,18 @@
   "name": "node-polyglot",
   "version": "2.3.0",
   "description": "Give your JavaScript the ability to speak many languages.",
-  "main": "index.js",
+  "main": "dist/polyglot.common-js.js",
+  "module": "dist/polyglot.es-modules.js",
   "scripts": {
-    "prepublish": "safe-publish-latest",
-    "pretest": "npm run --silent lint",
+    "build": "npm-run-all -p build:*",
```

Comment:
this level of indirection is not good; all npm run-scripts should be explicitly mentioned.